tests/models.py

```diff
@@ -70,5 +70,8 @@ def property_string(self):
 
 
 class BlogPost(models.Model):
-    author = models.ForeignKey(User)
+    author = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE
```

Comment:
What required this change?

Reply:
In Django>2, the `on_delete` argument on a `ForeignKey` is now required: https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0